src/lib/pickles/step_verifier.ml

```diff
@@ -833,7 +833,7 @@ struct
         (struct
           let constant_term = Plonk_checks.Scalars.Tick.constant_term
 
-          let index_terms = Plonk_checks.Scalars.Tick_with_lookup.index_terms
+          let index_terms = Plonk_checks.Scalars.Tick.index_terms
```

Comment:
Great! This answers and fixes a `FIXME?` that I have on another branch. 

We could thus use `Plonk_check.Scalars.Tick` directly here instead of the newly defined module by `struct ... end`, right?